helm-chart/amalthea-sessions/values.yaml

```diff
@@ -36,3 +36,16 @@ sidecars:
   image:
     repository: renku/sidecars
     tag: latest
+csi-rclone:
+  install: false
+  # This section is only relevant if you are installing csi-rclone as part of Amalthea.
+  # When this chart is deployed as part of Renkulab the following value should match the name of the storage class
+  # configured for Notebooks cloudstorage.
+  ## Name of the csi storage class to use for RClone/Cloudstorage. Should be unique per cluster.
+  # storageClassName: csi-rclone
+  # csiNodepluginRclone:
+  #   nodeSelector: {}
+  # Set tolerations if you have taints on your user session nodes. The csi has to run on every node
+  # where it is used.
+  #   tolerations: []
+  #   affinity: {}
```

Comment:
```suggestion
dependencies:
  csi-rclone: false
```

Reply:
I am not using the `dependencies` key as suggested because this is the array of child helm charts repositories.